### PR TITLE
Fix Kafka producer client configuration

### PR DESCRIPTION
**Pull request by indigo423**
Trivial change that clarifies how to configure the Kafka producer client which was reported in our discussion chat.

### CODE REVIEW

File: docs/modules/operation/pages/kafka-producer/enable-kafka.adoc [hunk 1]
```diff
@@ -6,20 +6,27 @@ The Kafka Producer is disabled by default and can be enabled as follows.
...
 ----
 
+.Configure the Kafka producer client
+[source, karaf]
+----
+config:edit org.opennms.features.kafka.producer.client
+config:property-set bootstrap.servers kafka-broker-ip:9092 <1>
+config:update
+----
+<1> Set the Kafka broker IP by replacing the kafka-broker-ip and port you want to connect.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/kafka-producer/enable-kafka.adoc:22]:
```suggestion
<1> Set the Kafka broker IP by replacing the kafka-broker-ip and port you want to connect to.
```

File: docs/modules/operation/pages/kafka-producer/enable-kafka.adoc [hunk 2]
```diff
@@ -6,20 +6,27 @@ The Kafka Producer is disabled by default and can be enabled as follows.
...
+    You can add multiple hosts in a comma separated list, e.g. `kafka-broker-1:9092,kafka-broker-2:9092`
```
Comment (Bonrob2) [inline docs/modules/operation/pages/kafka-producer/enable-kafka.adoc:23]:
```suggestion
    Add multiple hosts in a comma-separated list, for example, `kafka-broker-1:9092,kafka-broker-2:9092`
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
Minor changes.

File: docs/modules/operation/pages/kafka-producer/enable-kafka.adoc (hunk 1 above)
Comment (mmahacek) [reply docs/modules/operation/pages/kafka-producer/enable-kafka.adoc:22]: Is a preposition word we want to end a sentence with?

Comment (Bonrob2) [reply docs/modules/operation/pages/kafka-producer/enable-kafka.adoc:22]: It's not usually good practice, but in this situation it's okay. Changing it to "with which you want to connect" would be too awkward. Most style guides allow it now under these types of circumstances. 
We could say "... and connection port." if that would still make sense to users.

Comment (mmahacek) [reply docs/modules/operation/pages/kafka-producer/enable-kafka.adoc:22]: Cool cool cool

**Comment by indigo423** [comment]
@Bonrob2 thank you for the improvements!

**Review by mmahacek — APPROVED** [review]
While the highlighter.js doesn't have syntax rules for applying color coding to `karaf` source blocks, they do make it easier for the user to know where they should be running the commands, and removing the karaf prompt makes it easier to copy/paste.

LGTM!

**Comment by indigo423** [comment]
@Bonrob2 Just FYI, cause you requested changes for the PR, you're the only one with the power to unblock the merge if everything is as you expect :)

Screenshot 2021-11-17 at 00 19 45 1

**Review by Bonrob2 — APPROVED** [review]
LGTM!

**Pull request merged**